.travis.yml

```diff
@@ -11,18 +9,15 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
```

Comment:
this is because the config is wrong. It should be this:

```yaml
matrix:
    allow_failures:
        - php: hhvm
```

instead of what you used:

```yaml
matrix:
    allow_failures:
        - hhvm
```